install-dev/theme/views/configure.php

```diff
@@ -99,6 +99,21 @@
 		<?php echo $this->displayError('shop_timezone') ?>
 	</div>
 
+    <!-- Enable SSL -->
+    <div class="field clearfix">
+        <label class="aligned"><?php echo $this->translator->trans('Enable SSL', array(), 'Admin.Shopparameters.Feature'); ?></label>
```

Comment:
Shouldn't we localize this string in the _Install_ domain instead? I know it already exists in _Admin.Shopparameters.Feature_ but the installer part is rather separated from the core. What do you think, @Junebyun?
🤔

Reply:
:heavy_check_mark: Thanks @Junebyun & @LouiseBonnard